Anytype/Sources/PresentationLayer/TextEditor/AccessoryView/SlashMenu/Models/Actions/SlashActionMedia.swift

```diff
@@ -75,4 +75,13 @@ enum SlashActionMedia: CaseIterable {
             return .text(.code)
         }
     }
+    
+    var titleSynonyms: [String] {
+        switch self {
+        case .image:
+            return [Loc.picture]
+        default:
+            return []
+        }
+    }
```

Comment:
Let's populate this synonyms for another cases i suggest just to ask chatgpt for synonyms and pick top three